Promact.Oauth.Server/src/Promact.Oauth.Server/Controllers/ConsumerAppController.cs

```diff
@@ -86,24 +83,23 @@ public ConsumerAppController(IConsumerAppRepository iConsumerAppRepository, User
         * }
         */
         [HttpGet]
-        [Route("getConsumerApps")]
+        [Route("")]
         public async Task<IActionResult> GetConsumerApps()
         {
             try
             {
-                List<ConsumerApps> listOfApps = await _consumerAppRepository.GetListOfApps();
+                List<ConsumerApps> listOfApps = await _consumerAppRepository.GetListOfConsumerApps();
                 return Ok(listOfApps);
             }
-            catch (Exception ex)
+            catch (FailedToFetchDataException)
```

Comment:
I have not change anything in ConsumerAppController. but I know why we are use of this I think  any exception occurs or list of ConsumerApp are empty so end user show the message "list is empty".